website/docs/getting-started/emulating-docker-cli-with-podman.md

```diff
@@ -0,0 +1,45 @@
+---
+id: emulating-docker-cli-with-podman
+title: Emulating Docker CLI with Podman
+description: Emulation Docker CLI with Podman can make it easier to migrate from Docker to Podman, as it allows you to continue using familiar Docker commands while taking advantage of the benefits of Podman.
+tags: [podman-desktop, getting-started, cli,]
+keywords: [podman desktop, podman, containers]
+---
+
+# Emulating Docker CLI with Podman
+
+Consider emulating Docker CLI with Podman to migrate transparently to Podman.
+
+* Continue using familiar Docker commands.
+* Take advantage of the benefits of Podman on macOS.
+* Your tools, such as [Maven](https://maven.apache.org/) or [Testcontainers](https://www.testcontainers.org/), communicate with Podman without reconfiguration.
+
+#### Prerequisites
+
+* Podman
+
+#### Procedure
+
+1. Create a `/usr/bin/docker` script:
```

Comment:
Hi, shouldn't it be in `/usr/local/bin` ?

Reply:
For a user created script, it makes entirely sense.

```suggestion
1. Create a `/usr/local/bin/docker` script:
```

Note: I didn't think about it first. I took the script from the Fedora `podman-docker` package, which installs in `/usr/bin/`, the standard location for a package.